frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -34,6 +34,13 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
+type BuildAPICredentials struct {
+	AccessToken string
+	// The credential for exchanging access tokens should be generated from a GCP project that
+	// enables the Build API. If it isn't, CredsUserProjectID is required for successful API usage.
+	CredsUserProjectID string
```

Comment:
nit: `UserProjectID string`, the prefix `Creds` in `CredsUserProjectID` is redundant.